src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2022,16 +2023,40 @@ private IOperation PeekOperand()
             return visitedArray;
         }
 
-        private ImmutableArray<IArgumentOperation> VisitArguments(ImmutableArray<IArgumentOperation> arguments)
+        private ImmutableArray<IArgumentOperation> VisitArguments(ImmutableArray<IArgumentOperation> arguments, bool instancePushed)
         {
-            return VisitArray(arguments, UnwrapArgument, RewriteArgumentFromArray);
+            VisitAndPushArguments(arguments, instancePushed);
+
+            var visitedArguments = PopArray(arguments, RewriteArgumentFromArray);
+            return visitedArguments;
+        }
+
+        private void VisitAndPushArguments(ImmutableArray<IArgumentOperation> arguments, bool instancePushed)
+        {
+            var previousInterpolatedStringHandlerContext = _currentInterpolatedStringHandlerContext;
+            if (arguments.WhereAsArray(arg => arg.Value is IInterpolatedStringHandlerCreationOperation) is { IsDefaultOrEmpty: false } interpolatedStrings)
+            {
+                _currentInterpolatedStringHandlerContext = new InterpolatedStringHandlerContext(
+                    interpolatedStrings,
```

Comment:
>interpolatedStrings [](http://example.com/codeflow?start=20&length=19)

I do not see a benefit in recording ```IArgumentOperation```s here instead of ```IInterpolatedStringHandlerCreationOperation```s